modules/installation-ibm-cloud-configure-vpc-for-endpoint-gateways.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_cloud_public/installing-ibm-cloud-restricted.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-ibm-cloud-configure-vpc-for-endpoint-gateways_{context}"]
+= Allowing endpoint gateway traffic
+
+If you are using {ibm-cloud-name} Virtual Private endpoints, your Virtual Private Cloud (VPC) must be configured to allow traffic to and from the endpoint gateways.
+
+A VPC's default security group is configured to allow all outbound traffic to endpoint gateways. Therefore, the simplest way to allow traffic between your VPC and endpoint gateways is to modify the default security group to allow inbound traffic on port 443.
+
+[NOTE]
+====
+If you choose to configure a new security group, the security group must be configured to allow both inbound and outbound traffic.
+====
+
+.Prerequisites
+
+* You have installed the {ibm-cloud-name} Command Line Interface (`ibmcloud`).
```

Comment:
```suggestion
* You have installed the {ibm-cloud-name} Command Line Interface utility (`ibmcloud`).
```
I'm not sure if this is new content in this PR or reused from other areas.

Reply:
Net new. I updated it. Thanks.